ios/FluentUI/Core/Theme/Tokens/TokenizedControlView.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+/// SwiftUI-specific extension to `TokenizedControl`.
+protocol TokenizedControlView: TokenizedControlInternal {
+    /// Modifier function that updates the design tokens for a given control.
+    ///
+    /// - Parameter tokens: The tokens to apply to this control.
+    ///
+    /// - Returns: A version of this control with these overridden tokens applied.
+    func overrideTokens(_ overrideTokens: TokenType?) -> Self
+}
+
+// MARK: - Extensions
+
+extension TokenizedControlView {
+    /// Modifier function that updates the design tokens for a given control.
+    ///
+    /// - Parameter tokens: The tokens to apply to this control.
+    ///
+    /// - Returns: A version of this control with these overridden tokens applied.
+    func overrideTokens(_ overrideTokens: TokenType?) -> Self {
```

Comment:
I like this! 
It provides the default implementation to save all SwiftUI controls from implementing the same modifier while not requiring UIKit controls to have to implement a func that looks like a modifier.

Reply:
Yep! This has been bothering me since the first version of "new" tokens, so I figured now was the time to fix it.